packages/cli/src/commands/link.ts

```diff
@@ -20,14 +20,13 @@ const register: (program: any) => any = (program) => program
 
 async function action(dependencies: string[], options: any): Promise<void> {
   const { install, forceInstall, interactive } = options;
-  const installDependencies = forceInstall || (install && interactive ? undefined : install);
+  const installDependencies = install || forceInstall;
```

Comment:
Where does `forceInstall` come from?

Reply:
From the `create` command.  If a user runs a `zos create openzeppelin-eth/SomeContract` without having `openzeppelin-eth` linked, zos will install and link it for them.